ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -1101,7 +1103,11 @@ tflite_checkAvailability (accl_hw hw)
   return -ENOENT;
 }
 
+#if TFLITE_VERSION == 1
 static gchar filter_subplugin_tensorflow_lite[] = "tensorflow-lite";
+#else
+static gchar filter_subplugin_tensorflow_lite[] = "tensorflow2-lite";
+#endif
```

Comment:
WHat if both tf2-lite and tf1-lite are installed simultaneously?

Reply:
I think it's okay because even if they share the same source files, different .so files (and different names) will be created.